.storybook/main.js

```diff
@@ -28,7 +28,6 @@ module.exports = {
       },
     },
     "@storybook/addon-controls",
-    "@storybook/addon-queryparams",
```

Comment:
I thought the query params were going to be viable again after moving away from the enums.